package.json

```diff
@@ -41,6 +41,7 @@
   ],
   "dependencies": {
     "can-autoplay": "^3.0.0",
+    "cryptiles": "^4.1.2",
```

Comment:
Is this now required because of the previous dependency updates? If so, is there a test that should have caught that?

Reply:
I ran the 'npm audit' command and it claims it fixed all of the vulnerabilities, then this still proved to be an issue. I'll investigate if there's a test we could add to the repo.

Reply:
I don't see anything that hooks into this as a test we can run before committing.